source/_patterns/02-organisms/60-checkout/11-donation-item.mustache

```diff
@@ -0,0 +1,18 @@
+<div class="loan-row row small-collapse">
+	<div class="loan-image small-2 columns">
+		<img height="150" width="150" title="Donation to Kiva" src="{{imageurl}}">
```

Comment:
width/height should be set in the css instead of the html

Reply:
oops getting that outta there